studio/schemas/carpark.js

```diff
@@ -33,10 +38,10 @@ export default {
       },
     },
     {
-      name: 'categories',
-      title: 'Categories',
+      name: 'subTag',
```

Comment:
I am not sure what the definition of category is, the example is 商場. I think our use case at this point would all be shopping malls, therefore, it think tags and subtags might be more useful.
which may include
商場類別 as tag and 寵物友善 as subtag

Reply:
Tag is a better wording for category, as a carpark / even a post can be associated to multiple tags
I think tags is enough, it can be an array with anything and then we handle the logic (eg. list out all carparks with the same tag)